dateparser/languages/locale.py

```diff
@@ -184,10 +185,28 @@ def translate_search(self, search_string, settings=None):
             original_tokens, simplified_tokens = self._simplify_split_align(sentence, settings=settings)
             translated_chunk = []
             original_chunk = []
+            simplified_tokens_length = len(simplified_tokens)
```

Comment:
We could put the `-1` here :)

```suggestion
            last_token_index = len(simplified_tokens) - 1
```

In that way the expression below will be easier to read.

Reply:
Done 👍